src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.ts

```diff
@@ -356,8 +357,10 @@ export class InstructorCourseEnrollPageComponent implements OnInit {
   getCourseEnrollPageData(courseid: string): void {
     this.existingStudents = [];
     this.hasLoadingStudentsFailed = false;
+    this.isLoading = true;
     this.courseService.hasResponsesForCourse(courseid).subscribe((resp: HasResponses) => {
       this.coursePresent = true;
+      this.isLoading = false;
```

Comment:
The statement can go to the complete callback (the third argument of `.subscribe` function) to reduce the redundancy in both success and error case .

Reply:
Thank you, will do that